app/main.py

```diff
@@ -1,2 +1,61 @@
+from dataclasses import dataclass
+from typing import Hashable
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: any
+    hash_value: int
+
+
 class Dictionary:
-    pass
+    initial_capacity = 8
+    threshold = 2 / 3
```

Comment:
The expression `2 / 3` results in a float value (approximately 0.6667). If the intention is to have an integer threshold, consider using integer division `2 // 3` or explicitly setting the threshold as a float if that is the intended behavior.